src/com/google/javascript/jscomp/AbstractCommandLineRunner.java

```diff
@@ -1105,7 +1206,7 @@ int processResults(Result result, List<JSModule> modules, B options)
     return SourceCodeEscapers.javascriptEscaper().asFunction();
   }
 
-  void outputSingleBinary() throws IOException {
+  void outputSingleBinary(B options) throws IOException {
     Function<String, String> escaper = null;
```

Comment:
No - if using json_streams, all JS and Sourcemaps will be streamed out. I refactored this method in a similar fashion to the note above regarding the helper method in module outputing.